Makefile

```diff
@@ -34,8 +34,8 @@ verify:
 install: monorepo install-treecreeper
 
 monorepo:
-	npm install ysugimoto/athloi
-	npx athloi exec --concurrency 10 -- npm install --no-package-lock
+	npm install @financial-times/athloi
+	npx athloi exec --concurrency 1 -- npm install --no-package-lock
```

Comment:
We should set the concurrency as 1 otherwise raise an error of resolving each package dependencies